tools/main/main.cpp

```diff
@@ -587,12 +587,12 @@ int main(int argc, char ** argv) {
 
                 if (n_past + (int) embd.size() >= n_ctx) {
                     if (!params.ctx_shift){
-                        LOG_DBG("\n\n%s: context full and context shift is disabled => stopping\n", __func__);
+                        LOG_INF("\n\n%s: context full and context shift is disabled => stopping\n", __func__);
                         break;
                     }
 
                     if (params.n_predict == -2) {
-                        LOG_DBG("\n\n%s: context full and n_predict == -%d => stopping\n", __func__, params.n_predict);
+                        LOG_INF("\n\n%s: context full and n_predict == -%d => stopping\n", __func__, params.n_predict);
```

Comment:
Remove extra minus `-`:

```suggestion
                        LOG_INF("\n\n%s: context full and n_predict == %d => stopping\n", __func__, params.n_predict);
```